index.html

```diff
@@ -0,0 +1,82 @@
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSK2rfCGbrDopSuC8R5mnnRymSX0r1mVkqQAEiwNh6mHMDNfQ4NTHlySOnrq7B_XZEjhlU&usqp=CAU" id="header-img">
+        <nav id="nav-bar">
+      
+            <ul>
+              <li><a href="#features" class="nav-link">Features</a></li>
+              <li><a href="#video" class="nav-link">video</a></li>
+              <li><a href="#prices" class="nav-link">Prices</a></li>
+              <li><a href="#form" class="nav-link">form</a></li>
+            </ul>
+        </nav>
+    </header>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+        <h2>Logitech G915 The best on the market</h2>
+        <input type="email" name="email" id="email" placeholder="Please enter your email" required>
+        <input type="submit" value="Submit" id="submit">
+    </form>
+    <section id="feature__section">
+        <div id="features">
+            <div class="feature__1">
+                <i id="icon1" class="bi bi-star-fill"></i>
+                <h3>Premium style</h3>
+                <p><strong>G915</strong> is a new class of Wireless mechanical gaming Keyboard with 3 selections of low-profile GL switches and pro-grade 1 ms LIGHTSPEED Wireless.</p>
+            </div>
+            <div class="feature__2">
+                <i id="icon2" class="bi bi-truck"></i>
+                <h3>Fast delivery</h3>
+                <p>Our delivery service will make sure that get the product as fast as we can.</p>
+            </div>
+            <div class="feature__3">
+                <i id="icon3" class="bi bi-journal"></i>
+                <h3>Best Quality</h3>
+                <p>The Logitech G915 is an exceptional keyboard for gaming. The low profile switches are incredibly responsive and provide nice tactile feedback, and the keyboard has extremely low latency.</p>
+            </div>
+        </div>
+    </section>
+
+    <section id="prices" >
+        <div>
+            <h3>TENOR TROMBONE</h3>
+            <p>$600</p>
+            <span>Lorem ipsum </span>
+            <span>Lorem ipsum </span>
+            <span>Lorem ipsum </span>
+
+            <a href="#">SELECT</a>
+        </div>
+        <div>
+            <h3>TENOR TROMBONE</h3>
+            <p>$600</p>
+            <span>Lorem ipsum </span>
+            <span>Lorem ipsum </span>
+            <span>Lorem ipsum </span>
+
+            <a href="#">SELECT</a>
+        </div>
+        <div>
+            <h3>TENOR TROMBONE</h3>
+            <p>$600</p>
+            <span>Lorem ipsum </span>
+            <span>Lorem ipsum </span>
+            <span>Lorem ipsum </span>
+
+            <a href="#">SELECT</a>
+        </div>
```

Comment:
- [ ] Kindly make sure to replace `div` tags with `article` tag for better accessibility and SEO.
- [ ] Kindly make sure to add `span` tags inside the unordered list element `ul` for accessibility purposes.
- [ ] Kindly make sure to remove empty lines as they make your file not clean.